Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -41,21 +41,37 @@ public class BTShopperInsightsClient {
             let postParameters = BTEligiblePaymentsRequest(
                 email: request.email,
                 phone: request.phone,
-                merchantID: "TODO-merchant-id-type"
+                merchantID: "MXSJ4F5BADVNS"
```

Comment:
note: adding a sample merchant id for now, but still need to figure how to fetch this value (tbd)